webapp/migrations/Version20221004135409.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DoctrineMigrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+final class Version20221004135409 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return 'Allow super lazy evaluation where nothing is judged unless requested.';
+    }
+
+    public function up(Schema $schema): void
+    {
+        $this->addSql('ALTER TABLE contestproblem CHANGE lazy_eval_results lazy_eval_results INT UNSIGNED DEFAULT NULL COMMENT \'Whether to do lazy evaluation for this problem; if set this overrides the global configuration setting\'');
```

Comment:
does this do the right thing on conversion? As in, I assume `true` gets changed to a `1`, and `false` to a `0`, right? It should probably be `2` and `1` instead?

Reply:
This migration indeed does not work anymore since I changed the constants. Will think about how to do this properly.

Reply:
Implemented the solution to convert the "false" to a 2 as the boolean yes (lazy is true) is already done correctly.